### ISSUE TITLE
mention scenario is made with animation-flag

### ISSUE THREAD

**Issue by alixander** [terrastruct/d2-docs#199]
https://d2lang.com/tour/scenarios/

in general should do so for all animations

**Comment by Sean10** [comment]
Yes. It's better to mention this flag. User need to render it with "--animate-interval int"
When using [D2 Playground](https://play.d2lang.com) , it's not default with this. It doesn't support rendering animation svg for preview.
